docs/installation.md

```diff
@@ -10,74 +10,53 @@ These steps are required once per machine.
-you will need to copy paste it on your machine.
+data are produced by an sbt plugin.
 
-Here's the source of the plugin:
-https://github.com/scalameta/metals/blob/master/project/MetalsPlugin.scala
+You can install the plugin with
 
-Copy the source to either (depending on your sbt version):
-
-* (sbt 0.13) `~/.sbt/0.13/plugins/MetalsPlugin.scala`
-* (sbt 1.+) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
-
-## Step 2 - build the VSCode extension
-
-The VSCode extension is not yet published on the Marketplace, so you'll need to
-build it locally.
-
-* Make sure you have installed `node`, `npm` and VS Code.
-* `cd vscode-extension`
-* `npm install`
-* `npm run build`
-* `code --install-extension metals-0.1.0.vsix`
-
-## Step 3 - publish the server locally
+```
+addSbtPlugin("org.scalameta" % "sbt-metals" % "<version>")
+```
 
-From the repo root run `sbt publishLocal`
+## Step 2 - VSCode extension
+The VSCode extension is published on the Marketplace. You can open VSCode and search for it.
 
 # Per-project setup
 
 These steps are required on each project.
 
-## Step 1 - add semanticdb-scalac compiler plugin to your project
+## Quick-start
+The quickest way to get started with Metals is to use the `metalsSetup` command in sbt.
+
+```
+sbt
+> metalsSetup
+```
 
-Some features like definition/references/hover rely on artifacts produced by a
-compiler plugin called `semanticdb-scalac`. There are two alternative ways to
-install `semanticdb-scalac`.
+The command will create the necessary metadata in the `.metals` directory
+(which you should not checkout into version control) and setup the `semanticdb-scalac` compiler
+plugin for the current sbt ession.
 
-The first option is to enable `semanticdb-scalac` permanently for your project
-in `build.sbt` with:
+Note that you will need to invoke `metalsSetup` (or `semanticdbEnable`) whenevery you close and
+re-open sbt. For a more persistent setup, keep reading.
 
-```scala
- addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.8" cross CrossVersion.full)
-scalacOptions += "-Yrangepos"
-```
+## Persisting the semanticdb-scalac compiler plugin
+Some features like definition/references/hover rely on artifacts produced by a compiler plugin
+called `semanticdb-scalac`.
 
-The second option is to enable `semanticdb-scalac` only for an active sbt
-session by running `semanticdbEnable` from the sbt shell.
+`metalsSetup` enables the plugin on the current session (by invoking `semanticdbEnable`), but you
+can choose to enable it permanently on your project by adding these two settings in your sbt build
+definition:
 
 ```scala
-$ sbt
-> semanticdbEnable // automatically runs libraryDependencies += compilerPlugin(...)
-> compile // re-compile project with semanticdb-scalac compiler plugin
-> ...
+addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.8" cross CrossVersion.full)
+scalacOptions += "-Yrangepos"
 ```
 
 As soon as you exit the sbt shell you need to re-run `semanticdbEnable` next
```

Comment:
isn't this in contrast with the rest of the section, as it stands now?

Reply:
Nice catch, I've missed it while copy-pasting stuff around.